### ISSUE TITLE
Create chapter for cfform

### ISSUE THREAD

**Issue by cfjedimaster** [cfjedimaster/ColdFusion-UI-the-Right-Way#14]

**Comment by savaticus** [comment]
Reading the example, I see a huge red flag. Only doing client side validation is the path to compromise. It's fine to do client side validation, but one also must do server side validation, because there is nothing stopping someone from bypassing your javascript when sending data to the server.

**Comment by cfjedimaster** [comment]
You are right - and we will gladly take a PR that adds the server side validation in.
